public/config.json

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
The fact that this file must exist and gets overwritten in the `build` dir complicates the development workflow as it means that developers have to overwrite it when developing with their own community maps or copy it after every single build if i see that correctly?

Imho the easiest  option here would be to exclude it from the deletion when wiping the build dir?

Reply:
This file should not be tracked by Git. I will remove it and update the docs. Like before you have to get a config file from somewhere.

Reply:
https://github.com/freifunk/meshviewer/compare/b113ad7e60984d59be888c2a520aff6749e067e5..8e06242e20cb7679ed27e5764434e557c2181244